게임 진행 화면 에러 해결

swjw13 opened the pull request:
## Related Issue

- resolved boostcampwm-2022/android11-almost-there#186
- resolved boostcampwm-2022/android11-almost-there#180
- resolved boostcampwm-2022/android11-almost-there#179
- resolved boostcampwm-2022/android11-almost-there#174
- resolved boostcampwm-2022/android11-almost-there#133


## 작업 내용 요약

- 게임 준비를 누르지 않은 유저가 맵에 보이지 않도록 구성
- TMap 마커 제거
- 약속 장소가 저장된 장소와 다른 에러 해결
- 유저마다 남은 시간이 다른 에러 해결

## Checklist

- [x] Merge 되는 브랜치가 올바른가?
- [x] Reviewers, Assignees, Labels, Projects, Milestone 설정을 했는가?
- [x] 스스로 코드 리뷰를 충분히 진행했는가?
- [x] 프로젝트의 스타일 가이드라인(컨벤션)을 준수하는가?

swjw13 committed:
:bug: fix: Tmap 설정 변경

- 하단 TMap 로고 제거
- 초기 위치를 현재위치를 가져올 수 있으면 현재 위치로, 없다면 네이버 1784로 설정
- 장소 입력에서 바로 확인을 누를 경우 검색 결과가 없다고 하는 버그 수정

Issue #133 #174

swjw13 committed:
:bug: fix: 입력 장소 에러 해결

- 설정한 장소와 장소 입력 재입장시에 나타나는 위치가 다른 버그 수정

Issue #180

swjw13 committed:
:bug: fix: 게임 진행 화면 에러 해결

- 레디를 하지 않은 유저는 화면에 띄우지 않도록 변경
- Bottom Sheet에서 모든 유저들의 남은 시간이 동일하게 나타나던 오류 수정
- 기타 게임 예외처리 고려

Issue #179 #186

swjw13 committed:
Merge remote-tracking branch 'upstream/develop' into bug/promise-location

fbghgus123 commented on firebase/src/main/kotlin/com/woory/firebase/datasource/DefaultFirebaseDataSource.kt:
```diff
@@ -59,6 +60,32 @@ class DefaultFirebaseDataSource @Inject constructor(
 
+    override suspend fun getReadyUserList(code: String): Result<List<UserModel>> =
+        withContext(scope.coroutineContext) {
+            val result = kotlin.runCatching {
```
kotlin 을 삭제했으면 좋겠습니다 !

fbghgus123 commented on presentation/src/main/kotlin/com/woory/presentation/ui/gaming/GamingActivity.kt:
```diff
@@ -44,8 +44,14 @@ class GamingActivity : BaseActivity<ActivityGameResultBinding>(R.layout.activity
         super.onCreate(savedInstanceState)
+
+        if (gameCode == null) {
+            showSnackBar(binding.root, "참여 코드가 없습니다.")
```
string resource에 넣어주면 좋을 것 같아요

fbghgus123 reviewed (approved):
고생하셨습니다 !

swjw13 committed:
:bug: fix: 게임 진행 화면 에러 해결

- 자기장 줄어드는 속도가 작은 버그 해결
- Bottom Sheet 남은 시간이 로딩이 느린 버그 해결
- 게임 화면에서 초기 위치가 자신의 위치가 되도록 수정

Issue

swjw13 committed:
:bug: fix: 코드 리뷰 반영

- hard coding을 string resource로 관리
- 코드리뷰 반영

Issue 179

soulplay95 reviewed (approved):
고생하셨습니다~

swjw13 committed:
:bug: fix: 자기장 반지름이 10000으로 설정되는 버그 수정

- 자기장 Radius 초깃값 설정

Pull Request #188

tnwlsgk1002 reviewed (approved):
수고하셨습니다!
